lib/camera/index.js

```diff
@@ -10,8 +10,8 @@ var defaultOpts = {
   // input: '0',
   // codec: '.jpg',
   // frameCallback: function (err, frameRaw, faceDetected) {},
-  // onFrame: function (err, frameRaw) {},
-  // onFaceDetected: function (err, frameRaw) {},
+  // onFrame: function (frameRaw) {},
+  // onFaceDetected: function (frameRaw) {},
```

Comment:
Why are all these defaults commented out?

Reply:
to serve as documentation.
I added also the JSdoc above so in theory I can remove those